sycl/test-e2e/Matrix/element_wise_all_ops.cpp

```diff
@@ -10,6 +10,8 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
+// XFAIL: gpu-intel-dg2
+
```

Comment:
Same as abobe:
Adding XFAIL will make the code fail silently for the other supported combination.